File: Codopy/lab7 PM/lab06.py
#triangle check
#input
a = int(input())
b = int(input())
c = int(input())

#process
if (a > 0 and b > 0 and c > 0):
    if a != b and b != c:
        if (a > c) and (a > b):
            x = a**2
            y = c**2 + b**2
        elif b > a and b > c:
            x = b**2
            y = c**2 + a**2
        else:
            x = c**2
            y = a**2 + b**2
#output
print(x==y)

File: CodoC++/Lab02 If Else/Lab06.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    //input
    char sam = getchar();
    //process
    if ('a'<= sam && sam <= 'z'){
        printf("Output: lower case ");
    }else if ('A'<= sam && sam <= 'Z'){
        printf("Output: upper case ");
    }else if ('0' <= sam && sam <= '9'){
        printf("Output: digit");
    }else{
        printf("Output: others");
    }
}